position/__TEST_0_INIT_VARS_HELPERS_test.gno

```diff
@@ -0,0 +1,69 @@
+package position
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/r/gnoswap/v1/consts"
+
+	"gno.land/r/gnoswap/v1/gnft"
+
+	pl "gno.land/r/gnoswap/v1/pool"
+)
+
+var (
+	admin std.Address = consts.ADMIN
+	test1 std.Address = std.Address("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5")
+
+	fooPath string = "gno.land/r/onbloc/foo"
+	barPath string = "gno.land/r/onbloc/bar"
+	bazPath string = "gno.land/r/onbloc/baz"
+	quxPath string = "gno.land/r/onbloc/qux"
+
+	oblPath string = "gno.land/r/onbloc/obl"
+	// wugnotPath string = "gno.land/r/demo/wugnot" // from consts
+	// gnsPath string = "gno.land/r/gnoswap/v1/gns" // from consts
```

Comment:
Leftover?

Reply:
It's just comment for where we're getting those two tokens path, unlikely others declared in that file.

Should we remove entire line?

Reply:
it will be fine